easy-ui-tokens/src/shadow/base.json

```diff
@@ -20,6 +20,15 @@
     },
     "stuck-from-bottom": {
       "value": "0px -4px 4px 0px hsla(221, 30%, 74%, 0.25)"
+    },
+    "level.1": {
+      "value": "0px 1px 8px 0px hsla(226, 28%, 39%, 0.25)"
+    },
+    "level.2": {
+      "value": "0px 1px 10px 0px hsla(226, 28%, 39%, 0.4)"
+    },
+    "level.3": {
+      "value": "0px 1px 12px 0px hsla(226, 28%, 39%, 0.6)"
```

Comment:
**question**: i might be missing something very obvious here, but are these values intended to represent some of the new shadows for the cards in the ecom/nexus designs? if so, in the mockup i see slightly different values for the shadows (unless its documented somewhere else more precisely)


![Screenshot 2024-10-15 at 11 50 45 AM](https://github.com/user-attachments/assets/3303dc47-641c-41a7-a2e4-0926e0f0590b)